code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -0,0 +1,250 @@
+
+/obj/item/book/granter/martial/starter_sith
+	martial = /datum/martial_art/starter_sith
+	name = "Sith Sacred Texts (Vol. 2)"
+	martialname = "Sith Starter Secrets"
+	desc = "This book seems to crackle with electric malevolence. It depicts an old robed weirdo flinging pods at a peculiar green midget and fighting with an energy sword."
+	greet = "<span class='sciradio'>You have learned some of the ancient mysteries of the Sith! You can now move objects with the power of the Dark Side and deflect projectiles with energy swords.</span>"
+	icon_state ="bookcharge"
+	remarks = list("Oh, I'm afraid the force lightning will be quite operational when security arrives...","UNNNNNNNLIMITED POWAH!!!", "Let the hate flow through me...", "GooOood... GOOooOod...", "All is proceeding exactly as I have foreseen.", "The Dark Side is a pathway to many abilities some consider to be... unnatural...", "They will pay the price for their lack of vision...", "Execute Order 66...", "It seems it is me who is mistaken about a great. Many. Things.", "This space station is nothing compared to the power of the Dark Side...", "The tragedy of Darth Plaguis the Wise is not a tale the Jedi would tell me...", "Only now, at the end, do I understand.")
+
+/obj/item/book/granter/martial/starter_sith/onlearned(mob/living/carbon/user)
+	..()
+	sith_one_use(user)
+
+
+/datum/martial_art/starter_sith
+	name = "Sith Starter Secrets"
+	id = MARTIALART_STARTERSITH
+	allow_temp_override = FALSE
+	help_verb = /mob/living/carbon/human/proc/starter_sith_help
+	var/mob/living/carbon/human/owner
+
+
+/mob/living/carbon/human/proc/starter_sith_help()
+	set name = "Recall Sith Teachings"
+	set desc = "Remember the ancient secrets of the Sith."
+	set category = "Sith Starter Kit"
+
+	to_chat(usr, "<b><i>You retreat inward and recall the teachings of the Sith...</i></b>")
+
+	to_chat(usr, "<span class='notice'>Force Lightning</span>: Fires a bolt of force lightning at a random target within 3 tiles.")
+	to_chat(usr, "<span class='notice'>Force Push</span>: Move things around with the power of the Force.")
+	to_chat(usr, "<span class='notice'>Light Saber Training</span>: You can deflect most projectiles and melee attacks while holding an activated energy sword.")
+
+
+///Triggers on COMSIG_MOB_ATTACK_RANGED. Usually handles stuff like picking up items at range.
+/datum/martial_art/starter_sith/proc/on_ranged_attack(datum/source, atom/target)
+	if(owner.dna.check_mutation(TK)) //FULPSTATION Chaplain Starter Sith PR by Surrealistik Jan 2020; no stacking with Telekinesis
+		return
+	target.attack_tk(owner)
+
+/datum/martial_art/starter_sith/teach(mob/living/carbon/human/H, make_temporary = FALSE)
+	. = ..()
+	if(!.)
+		return
+	owner = H
+	RegisterSignal(H, COMSIG_MOB_ATTACK_RANGED, .proc/on_ranged_attack)
+
+
+/datum/martial_art/starter_sith/on_remove(mob/living/carbon/human/H)
+	. = ..()
+	UnregisterSignal(H, COMSIG_MOB_ATTACK_RANGED)
+
+/obj/item/melee/transforming/energy/sword/saber/sith

```

Comment:
Just use a regular red esword, jeeze. Why can muggles deflect projectiles with normal eswords, but not a sith one? There's no reason to make a special snowflake sword to make SURE that a muggle will never be able to deflect projectiles with THIS SPECIFIC KIND OF ESWORD, especially since they can still just make normal melee attacks with it.

Plus, having your own weapons/powers able to be turned against you is totally in-theme for a Palpatine reference (see: how Palpatine was defeated three times in the movies).

Reply:
Blocking works as normal with  the Sith sword; the one difference is that someone with Sith training has a 30% chance to _reflect_ projectiles. There are also other qualities the Sith sword has that requires a subtype.